Chinese/foufou/foufou.md

```diff
@@ -0,0 +1,3 @@
+犹言不是不是。多用于应对。《史记·太史公自序》：“‘夫子所论，欲以何明？’ 太史公 曰：‘唯唯，否否，不然。’” 清 戴名世
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.